portal/src/main/scala/cromwell/pipeline/CromwellPipelineApp.scala

```diff
@@ -25,7 +25,9 @@ object CromwellPipelineApp extends App {
 
   pipelineDatabaseEngine.updateSchema()
 
-  val route = authController.route ~ securityDirective.authenticated { userController.route }
+  val route = authController.route ~
+    securityDirective.authenticated { userController.route } ~
+    securityDirective.authenticated { projectController.route }
```

Comment:
Is it possible to nest `userController` route and `projectController` route inside one `securityDirective`?

Reply:
decided not to fix it in that pull request. Found out that securityDirectice.authenticated seems to make all routes require auth. It's worth to investigate further